test/unit/test_path.py

```diff
@@ -0,0 +1,84 @@
+from enum import Enum
+from unittest.mock import (
+    patch,
+)
+
+import pytest
+
+from exasol.bucketfs._path import (
+    infer_backend,
+    infer_path,
+)
+
+
+class StorageBackend(Enum):

```

Comment:
As @ahsimb said: This probably could go to production code.

Reply:
Or simply use [StorageBackend](https://github.com/exasol/bucketfs-python/blob/main/exasol/bucketfs/_path.py#L36), maybe making it public by importing in `bucketfs/__init__.py`.